Scripts/Mend SCA/get-library-ages.py

```diff
@@ -27,73 +28,81 @@
 
 ******** Usage ********
 Make sure to install the appropriate dependencies before running this script and set all required environment variables. Afterwards, you can run the script with:
-python3 get-library-ages.py
+pip3 install requests python-dateutil
+python3 get-library-ages.py -o <output_file>.json -a <max_age_in_days>
 
-Optional Arguments: the only optional argument for this script is WS_API_KEY. If this is not specified then the Mend API will log in to the last organization accessed from the Mend Platform.
+Optional Arguments: the only optional arguments for this script are WS_API_KEY and MEND_PRODUCT_TOKEN. 
+If WS_API_KEY is not specified then the Mend API will log in to the last organization accessed from the Mend Platform.
+If MEND_PRODUCT_TOKEN is not specified then it will pull everything in the organizaiton.
 
 Pre-requisites:
-apt-get install python39
+apt-get install python3.9
 pip install requests python-dateutil
 export MEND_USER_KEY='<MEND_USER_KEY>'
 export MEND_URL='<MEND_URL>
 export MEND_EMAIL='<MEND_EMAIL>'
-export OUTPUT_FILE='<OUTPUT_FILE>.json'
-export MAX_LIBRARY_AGE='<MAX_AGE_IN_DAYS>'
 export WS_API_KEY='<WS_API_KEY>' <-- Optional
+export MEND_PRODUCT_TOKEN='<PRODUCT_TOKEN>' <-- Optional
```

Comment:
add a note on how to get productUUID from https://docs.mend.io/bundle/mend-api-2-0/page/index.html#tag/Entities-Organization/operation/getDomainEntities